doc/model_details/HillTononi.ipynb

```diff
@@ -0,0 +1,792 @@
+    "\n",
+    "\\begin{equation}\n",
+    "\\frac{\\text{d}V}{\\text{d}t} = \\frac{-g_{\\text{NaL}}(V-E_{\\text{Na}})\n",
+    "-g_{\\text{KL}}(V-E_{\\text{K}})+I_{\\text{syn}}+I_{\\text{int}}}{\\tau_{\\text{m}}}\n",
+    "-\\frac{g_{\\text{spike}}(V-E_{\\text{K}})}{\\tau_{\\text{spike}}}\n",
+    "\\end{equation}\n",
+    "\n",
+    "- The equation does not contain membrane capacitance. As a side-effect, all conductances are dimensionless.\n",
+    "- Na and K leak conductances $g_{\\text{NaL}}$ and $g_{\\text{KL}}$ are constant, although $g_{\\text{KL}}$ may be adjusted on slow time scales to mimic neuromodulatory effects.\n",
+    "- Reversal potentials are assumed constant.\n",
+    "- Synaptic currents $I_{\\text{syn}}$ and intrinsic currents $I_{\\text{int}}$ are discussed below. In contrast to the paper, they are shown with positive sign here (just change in notation).\n",
+    "- The last term is a re-polarizing current only active during the refractory period, see below. Note that it has a different (faster) time constant than the other currents. It might have been more natural to use the same time constant for all currents and instead adjust $g_{\\text{spike}}$. We follow the original approach here."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Threshold, Spike generation and refractory effects\n",
+    "\n",
+    "The threshold evolves according to [HT05, p 1677]\n",
+    "\n",
+    "\\begin{equation}\n",
+    "\\frac{\\text{d}\\theta}{\\text{d}t} = -\\frac{\\theta-\\theta_{\\text{eq}}}{\\tau_{\\theta}}\n",
+    "\\end{equation}\n",
+    "\n",
+    "The neuron emits a single spike if \n",
+    "- it is not refractory\n",
+    "- membrane potential crosses the threshold, $V\\geq\\theta$\n",
+    "\n",
+    "Upon spike emission,\n",
+    "- $V \\leftarrow E_{\\text{Na}}$\n",
+    "- $\\theta \\leftarrow E_{\\text{Na}}$\n",
+    "- the neuron becomes refractory for time $t_{\\text{spike}}$ (`t_ref` in NEST)\n",
+    "\n",
+    "The repolarizing current is active during, and only during the refractory period:\n",
+    "\\begin{equation}\n",
+    "g_{\\text{spike}} = \\begin{cases}  1  & \\text{neuron is refractory}\\\\\n",
+    " 0 & \\text{else} \\end{cases}\n",
+    "\\end{equation}\n",
+    "\n",
+    "During the refractory period, the neuron cannot fire new spikes, but all state variables evolve freely, nothing is clamped. \n",
+    "\n",
+    "The model of spiking and refractoriness is based on Synthesis model `PulseIntegrateAndFire`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Intrinsic currents\n",
+    "\n",
+    "Note that not all intrinsic currents are active in all populations of the network model presented in [HT05, p1678f].\n",
+    "\n",
+    "Intrinsic currents are based on the Hodgkin-Huxley description, i.e.,\n",
+    "\n",
+    "\\begin{align}\n",
+    "I_X &= g_{\\text{peak}, X} m_X(V, t)^N_X h_X(V, t)(V-E_X) \\\\\n",
```

Comment:
I would prefer it if the meaning of the variables were stated explicitly. But maybe this is not necessary.